sippy-ng/src/App.js

```diff
@@ -39,7 +40,7 @@ import VariantStatus from './jobs/VariantStatus'
 
 const drawerWidth = 240
 
-const useStyles = makeStyles((theme) => ({
+export const useStyles = makeStyles((theme) => ({
```

Comment:
Are you reusing this for the toolbar in component readiness? I'd duplicate it instead so we can style them differently

Reply:
sounds good.